prime-router/src/test/kotlin/fhirengine/utils/FhirTranscoderTests.kt

```diff
@@ -71,4 +75,47 @@ class FhirTranscoderTests {
         assertThat(decodedBundle).isNotNull()
         assertThat(decodedBundle).equals(testBundle)
```

Comment:
Wow! Can you please open a new issue to fix this. There are 5 instances of .equals use with assertTHat and it needs to be isEqualTo. Otherwise these tests do not test anything.